openprescribing/dmd2/forms.py

```diff
@@ -21,7 +21,7 @@
 
 class SearchForm(forms.Form):
     q = forms.CharField(
-        label='Query or SNOMED code',
+        label='Query, SNOMED code, or BNF code/prefix',
```

Comment:
The BNF prefix will only work from the chemical level, i.e. 9 chars and longer, right?  We need to somehow make this clear - in the label, and perhaps in validation too?

Reply:
> The BNF prefix will only work from the chemical level, i.e. 9 chars and longer, right? 

No -- we just do `Q(bnf_code__startswith=q.upper())` in `DMDObjectQuerySet.search()`.

Currently we ensure that `q` is at least 3 characters long, but maybe it'd be sensible to make it longer.